articles/aks/troubleshooting.md

```diff
@@ -112,6 +112,7 @@ Follow the *Before you begin* steps in the appropriate doc to correctly register
 
 Naming restrictions are implemented by both the Azure platform and AKS. If a resource name or parameter breaks one of these restrictions, an error is returned that asks you provide a different input. The following common naming guidelines apply:
 
+* The hard limit for cluster name is 1-63 characters and can only contain letters, numbers, dash and underscore are allowed in middle.
```

Comment:
* Cluster names must be 1-63 characters. The only allowed characters are letters, numbers, dashes, and underscores. The first and last characters must be a letter or number.